packages/components/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- `ProgressBar`: Remove "experimental" designation for `ProgressBar`, add optional custom `width` prop ([#61062](https://github.com/WordPress/gutenberg/pull/61062)).
```

Comment:
Should we keep the "remove" experimental designation in the changelog or just list the `width` change?

Reply:
In this case, stabilizing/removing the experimental designation for this component is a very small change related to the storybook category, so I think we can add the API improvement as part of this PR, too. Also, by looking at other similar PRs, I can see that we shouldn't add the "remove experimental designation/stabilize" in the changelog, as it's considered an internal change. I'm happy to split, though. 

One question I have for @mirka is if we should also start exporting this component as part of this changeset to allow external users to start consuming it. I guess we should?

Reply:
> we should also start exporting this component as part of this changeset to allow external users to start consuming it

Oh, you're right. We need to remove the private API locking and export the component publicly.

FWIW I do agree with @tyxla that the width changes and the de-experimentalization are ideally separate PRs, if it isn't too much work.